<ui version="4.0" >
 <class>LogonWidgetClass</class>
 <widget class="QWidget" name="LogonWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>267</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QFrame" name="frame_" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>370</width>
     <height>249</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth" >
    <number>0</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topTitleFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="settingsButton" >
         <property name="font" >
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName" >
          <string>blueButton</string>
         </property>
         <property name="text" >
          <string>Settings</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forgotPasswordButton" >
         <property name="font" >
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName" >
          <string>blueButton</string>
         </property>
         <property name="text" >
          <string>RestorePassword</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="signUpButton" >
         <property name="font" >
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName" >
          <string>blueButton</string>
         </property>
         <property name="text" >
          <string>SignUp!</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="leftMargin" >
        <number>20</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>30</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout" >
         <property name="topMargin" >
          <number>2</number>
         </property>
         <property name="horizontalSpacing" >
          <number>10</number>
         </property>
         <property name="verticalSpacing" >
          <number>2</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="loginLabel" >
           <property name="font" >
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="accessibleName" >
            <string>whiteLabel</string>
           </property>
           <property name="text" >
            <string>Login</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy" >
            <cstring>loginCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="loginCombo" >
           <property name="editable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="passwordLabel" >
           <property name="font" >
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="accessibleName" >
            <string>whiteLabel</string>
           </property>
           <property name="text" >
            <string>Password</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy" >
            <cstring>passwordEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <layout class="QVBoxLayout" name="verticalLayout_4" >
           <property name="spacing" >
            <number>5</number>
           </property>
           <property name="topMargin" >
            <number>10</number>
           </property>
           <item>
            <widget class="QCheckBox" name="savePasswordCheck" >
             <property name="accessibleName" >
              <string>whiteLabel</string>
             </property>
             <property name="text" >
              <string>SavePasswordCheckBox</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoLogonCheckBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="accessibleName" >
              <string>whiteLabel</string>
             </property>
             <property name="text" >
              <string>AutoLoginCheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1" >
          <layout class="QVBoxLayout" name="verticalLayout_2" >
           <property name="topMargin" >
            <number>8</number>
           </property>
           <item>
            <widget class="QLineEdit" name="passwordEdit" >
             <property name="echoMode" >
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="bottomLinksFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <property name="spacing" >
        <number>30</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>10</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="signInButton" >
         <property name="font" >
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName" >
          <string>bigBlueButton</string>
         </property>
         <property name="text" >
          <string>SignIn</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>10</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="transBorder_4" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder" >
   <property name="geometry" >
    <rect>
     <x>386</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth" >
    <number>1</number>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>265</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <tabstops>
  <tabstop>loginCombo</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>savePasswordCheck</tabstop>
  <tabstop>autoLogonCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
